README.md

```diff
@@ -24,15 +24,15 @@ c = a;
 ### Your solution here:
 1.  What is `a`?
 ```
-a is ...
+a is ... 'bongos'
 ```
 2.  What is `b`?
 ```
-b is ...
+b is ... true 
 ```
 3.  What is `c`?
 ```
-c is ...
+c is ... 1
```

Comment:
c is 'bongos'

because c = a after a was changed to bongos. remember that java script start from the top and goes down from there